doc/observable.md

```diff
@@ -267,7 +267,7 @@ Conclusion:
 
 ## Anatomy of an Observable
 
-Observables are **created** using `Observable.create` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer and Subscription.
+Observables are **created** using `new Observable` or a creation operator, are **subscribed** to with an Observer, **execute** to deliver `next` / `error` / `complete` notifications to the Observer, and their execution may be **disposed**. These four aspects are all encoded in an Observable instance, but some of these aspects are related to other types, like Observer and Subscription.
```

Comment:
Do you think this might be easier to read if it was in list format?

1. .....
2. .....
3. .....
4. .....

That way it is easier to identify the 4 aspects.

Reply:
I'm going to rewrite this whole page soon, so I don't want to get into the minutia of changes right now.